lib/ex_admin/ex_admin.ex

```diff
@@ -237,7 +210,7 @@ defmodule ExAdmin do
 
   @doc false
   def get_title_actions(%Plug.Conn{private: _private, path_info: path_info} = conn) do
-    defn = get_registered_by_controller_route(path_info)
+    defn = conn.assigns.defn
```

Comment:
The only example i can think of is if they add their own plugs like authentication or authorization. So, not probable, but possible
> On May 22, 2016, at 1:53 PM, Roman Smirnov <notifications@github.com> wrote:
> 
> In lib/ex_admin/ex_admin.ex <https://github.com/smpallen99/ex_admin/pull/127#discussion_r64153631>:
> 
> > @@ -237,7 +210,7 @@ defmodule ExAdmin do
> >  
> >    @doc false
> >    def get_title_actions(%Plug.Conn{private: _private, path_info: path_info} = conn) do
> > -    defn = get_registered_by_controller_route(path_info)
> > +    defn = conn.assigns.defn
> @smpallen99 <https://github.com/smpallen99> What is the purpose of namespaces here? We assign these keys to connections that handle requests to admin controllers. So, no interference with frontend-controllers is possible.
> As to ea_required, maybe required_fields would be more descriptive?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/smpallen99/ex_admin/pull/127/files/eacbed0b90cb209c2a288c851a8f211628fa8f07#r64153631>